kurtosis-devnet/espresso-eb.yaml

```diff
@@ -1,10 +1,8 @@
 optimism_package:
+  observability:
+    enabled: false
   altda_deploy_config:
-    use_altda: true
```

Comment:
We no longer hook into alt-da for our integration

Reply:
Why is it simpler/better not to rely on the alt-da feature?

Reply:
Alt-DA relies on channel manager, so it can't be posting to DA layer for every block